dbtools/pg.py

```diff
@@ -1745,6 +1745,19 @@ def create_simplified_matviews(
         )
         return simplified_matviews
 
+    def get_view_defintion(self, view: str, schema=str) -> str:
+        """Retrieve the definition of the passed view as text."""
+        view_def_sql = sql.SQL("SELECT pg_get_viewdef('{schema}.{matview}')").format(
+            schema=sql.SQL(schema),
+            matview=sql.SQL(view)

```

Comment:
might be better to rename `matview` to something like `view` to better indicate this is for views or matviews